locales/ja.json

```diff
@@ -537,6 +545,18 @@
         "plural": "無効なARIA属性名です: ${data.values}"
       }
     },
+    "braille-label-equivalent": {
+      "pass": "aria-braillelabelはアクセシブルなテキストがある要素に対して使用されています",
+      "fail": "aria-braillelabelがアクセシブルなテキストがない要素に対して使用されています",
+      "incomplete": "アクセシブルなテキストを算出することができません"
+    },
+    "braille-roledescription-equivalent": {
+      "pass": "aria-brailleroledescriptionはアクセシブルなテキストがある要素に対して使用されています",
```

Comment:
Looks like another issue with the original. PR with the fix here: https://github.com/dequelabs/axe-core/pull/4133 Could you update this to match @ma10?

Reply:
Done in commit d80a727b